lib/room.h

```diff
@@ -999,7 +999,7 @@ public Q_SLOTS:
 
     void newFileTransfer(QString id, QUrl localFile);
     void fileTransferProgress(QString id, qint64 progress, qint64 total);
-    void fileTransferCompleted(QString id, QUrl localFile, QUrl mxcUrl);
+    void fileTransferCompleted(QString id, QUrl localFile, QUrl mxcUrl, Omittable<EncryptedFile> encryptedFile = std::nullopt);
```

Comment:
`Quotient::none` means the same thing as `std::nullopt` but if, God forbid, `Omittable` gets additional state on top of `std::optional`, `none` will be updated to follow `Omittable` while `std::nullopt` may lead to under-initialisation.
```suggestion
    void fileTransferCompleted(QString id, QUrl localFile, QUrl mxcUrl, Omittable<EncryptedFile> encryptedFile = none);
```